### PR TITLE
Fix field names in Solutions App project

### PR DESCRIPTION
**Pull request by bjss-chris**
Activity: [AB#9815](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9815)
Task: [AB#11095](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/11095)

### CODE REVIEW

**Review by bjss-andrew — APPROVED** [review]
LGTM

File: src/NHSD.BuyingCatalogue.Solutions.API/Controllers/RoadmapController.cs [hunk 1]
```diff
@@ -50,6 +50,6 @@ public async Task<ActionResult> Get([Required] string id)
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         public async Task<ActionResult> Update([Required] string id, UpdateRoadmapViewModel model) =>
-            (await mediator.Send(new UpdateRoadmapCommand(id, model?.Summary))).ToActionResult();
+            (await mediator.Send(new UpdateRoadMapCommand(id, model?.Summary))).ToActionResult();
```
Comment (PhysicsJon) [inline src/NHSD.BuyingCatalogue.Solutions.API/Controllers/RoadmapController.cs:53]: UpdateRoadmapViewModel should be updated to provide consistency here

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Other than one change this looks OK, although I have reservations around splitting roadmap into 2 words

File: src/NHSD.BuyingCatalogue.Solutions.API/Controllers/RoadmapController.cs (hunk 1 above)
Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Solutions.API/Controllers/RoadmapController.cs:53]: I think both Oxford and Cambridge dictionaries define road map like that (as two discrete words). I think the only place I've seen it as a single word is in BAPI! 😃 I'll update the 'view model' – that naming needs fixing too but that can come later – class as well to be consistent.

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**